sparql11/data-sparql11/json-res/jsonres01.srj

```diff
@@ -17,12 +17,12 @@
       {
         "s": { "type": "uri" , "value": "http://example.org/s3" } ,
         "p": { "type": "uri" , "value": "http://example.org/p2" } ,
-        "o": { "datatype": "http://www.w3.org/2001/XMLSchema#string" , "type": "literal" , "value": "bar" }
+        "o": { "type": "literal" , "value": "bar" }
       } ,
       {
         "s": { "type": "uri" , "value": "http://example.org/s4" } ,
         "p": { "type": "uri" , "value": "http://example.org/p4" } ,
-        "o": { "datatype": "http://www.w3.org/2001/XMLSchema#integer" , "type": "literal" , "value": "4" }
+        "o": { "type": "literal" , "value": "4" }
```

Comment:
This introduces the implicit rule "if a literal value is a JSON number that looks like a `xsd:integer`, then it should be parsed as a `xsd:integer` literal". Is it intended? It is not specify anywhere at might knowledge.

If we want to introduce it, it might also be nice to add more tests to cover more cases (boolean, the numbers `0.1` (`xsd:decimal`?) `0.1e3` (`xsd:double`?), `0.` (??)...

Reply:
No, that's not intended. Thank you for catching that.

I missed it because `json-res` isn't run by `manifest-sparq11-query`.

`csv-tsv-res/csvtsv02.tsv` is damaged as well.

`manifest-all` contains remote tests, which need special setup and the entailment tests.

I wonder if it would be useful to have `manifest.ttl` that is the "all local" part -- query, update and the result set formats.

Reply:
The entailment tests includes one with complex OWL reasoning.

The idea of `manifest.ttl` is a single place for the majority of triplestores.

Or `manifest-results.ttl`.
Or both!

Reply:
Makes sense. +1 to `manifest.ttl` or something like `manifest-with-results.ttl`.

Reply:
I'll add it to the reorg.
I'm currently minded to have `manifest-results` and `manifest` includes `-query,`, `-update` and `-results`.